api/src/wfl/api/workloads.clj

```diff
@@ -64,13 +64,13 @@
   "Use transaction `tx` to load `workload`(s) with `project`."
   [tx project]
   (let [do-load   (partial load-workload-impl tx)]
-    (map do-load (jdbc/query tx ["SELECT * FROM workload WHERE project = ?" project]))))
+    (mapv do-load (jdbc/query tx ["SELECT * FROM workload WHERE project = ?" project]))))
 
 (defn load-workloads
   "Use transaction TX to load all known `workloads`"
   [tx]
   (let [do-load (partial load-workload-impl tx)]
-    (map do-load (jdbc/query tx ["SELECT * FROM workload"]))))
+    (mapv do-load (jdbc/query tx ["SELECT * FROM workload"]))))
```

Comment:
Why the `mapv`s?

Reply:
Ah, to force query completion within the transaction?

Reply:
Exactly - I could have added `doall`s everywhere but I think eager evaluation is used more commonly.